### DISCUSSION TITLE
Lost all Sensors/Entities

### DISCUSSION THREAD

**Discussion by davidmhornsby** [BottlecapDave/HomeAssistant-OctopusEnergy#474] [Q&A]
Hi All, 

Been using this excellent integration for ages, then I started having issues.

They seem to have occurred around about the same time I took delivery of my Home Mini, but sadly I cannot confirm becuase I didn't notice right away.

My all my sensors stopped working, and when I re added the octopus integration I only have 2 entities available....

Anyone got any ideas what I can do to get this back?

Thanks
![Screenshot 2023-10-16 at 15 40 49](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/4046938/017850a7-f3ae-43de-a5fc-9e409aa9d051)
![Screenshot 2023-10-16 at 15 40 43](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/4046938/c9644efa-b048-49e5-8393-77a662a77a77)
![Screenshot 2023-10-16 at 15 40 06](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/assets/4046938/b45aac3d-3829-43c4-8aec-0a10c140b561)

**Comment by BottlecapDave** [comment]
That suggests that it was unable to get meter information from your account, or none of your meters had an active tariff. I assume you haven't switched tariff recently? Could you follow [these instructions](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/blob/develop/_docs/faq.md#ive-been-asked-for-my-meter-information-in-a-bug-request-how-do-i-obtain-this) and paste the results?

**Reply by davidmhornsby** [reply]
I reinstalled the integration and retsrted and the sensors cme back! thank you for taking time to reply!

**Comment by nicwats** [comment]
Sorry to hijack a thread but I'm having the exact same issue. It stopped working a while back so I did a fresh install and it's now only showing the same two entites as the OP. I tried following the instructions you mentioned above but there are no devices showing relating to octopus, just the two entities and the integration. I've tried a reinstall and re-start a couple of times but no luck.

Thanks, Nic

**Reply by BottlecapDave** [reply]
Do you have any errors in your logs?

**Reply by nicwats** [reply]
Looks like it's to do with the meters but not sure what (have XXX'd the serial numbers etc)...

First occurred: 14:50:36 (51 occurrences)
Last logged: 15:54:48

octopus_energy: Not adding entity with invalid device info: Invalid device info {'default_name': 'Electricity Meter', 'identifiers': {('octopus_energy', 'electricity_XXXXXXXXX_XXXXXXXX')}} for 'octopus_energy' config entry: device info needs to either describe a device, link to existing device or provide extra information.
octopus_energy: Not adding entity with invalid device info: Invalid device info {'default_name': 'Electricity Meter', 'identifiers': {('octopus_energy', 'electricity_XXXXXXX_XXXXXXXX')}} for 'octopus_energy' config entry: device info needs to either describe a device, link to existing device or provide extra information.
octopus_energy: Not adding entity with invalid device info: Invalid device info {'default_name': 'Gas Meter', 'identifiers': {('octopus_energy', 'electricity_XXXXXXXXXX_XXXXXXXXXX')}} for 'octopus_energy' config entry: device info needs to either describe a device, link to existing device or provide extra information.

**Reply by BottlecapDave** [reply]
That suggests you've updated your home assistant instance, but are not on the latest version of the integration. You'll either need to go into HACS to install the update (if done that way) or install the latest release from GitHub releases manually.

**Reply by nicwats** [reply]
Good shout, I'm being a donut!
